scheduler/kubernetes/encrypted-volumes.md

```diff
@@ -0,0 +1,176 @@
+---
+layout: page
+title: "Encrypted volumes using PVC"
+keywords: portworx, container, kubernetes, storage, k8s, flexvol, pv, persistent disk, encryption, pvc
+sidebar: home_sidebar
+meta-description: "Looking to use a encrypted volume with Kubernetes? Follow this step-by-step tutorial on how to provision encrypted volumes with Kubernetes."
+---
+
+* TOC
+{:toc}
+
+This document describes how to provision an encrypted volume using Kubernetes and Portworx. For more information on encryption refer the [Encrypted Volumes page](/manage/encrypted-volumes.html).
```

Comment:
As discussed offline, let's refactor the page into multiple pages with the first page guiding users what approach they should be following.